Mathlib/SetTheory/Cardinal/Finite.lean

```diff
@@ -172,6 +180,9 @@ theorem card_pLift (α : Type*) : card (PLift α) = card α :=
   card_congr Equiv.plift
 #align part_enat.card_plift PartENat.card_pLift
 
+lemma card_mono {s t : Set α} (ht : t.Finite) (h : s ⊆ t) : Nat.card s ≤ Nat.card t :=
```

Comment:
Can you please generalize this to injective functions?

Reply:
What statement are you thinking of?